File: data/derived_cot/rq2_eval/translation/qwen/_converted_code/java_to_cpp/AreaCalculator.cpp
#include <cmath>
#include <iostream>

class AreaCalculator {
private:
    double radius;

public:
    explicit AreaCalculator(double radius) : radius(radius) {}

    double calculate_circle_area() const {
        return M_PI * std::pow(radius, 2);
    }

    double calculate_sphere_area() const {
        return 4 * M_PI * std::pow(radius, 2);
    }

    double calculate_cylinder_area(double height) const {
        return 2 * M_PI * radius * (radius + height);
    }

    double calculate_sector_area(double angle) const {
        return std::pow(radius, 2) * angle / 2;
    }

    double calculate_annulus_area(double innerRadius, double outerRadius) const {
        return M_PI * (std::pow(outerRadius, 2) - std::pow(innerRadius, 2));
    }
};

int main() {
    AreaCalculator areaCalculator(2);

    std::cout << "Circle Area: " << areaCalculator.calculate_circle_area() << std::endl;
    std::cout << "Sphere Area: " << areaCalculator.calculate_sphere_area() << std::endl;
    std::cout << "Cylinder Area: " << areaCalculator.calculate_cylinder_area(3) << std::endl;
    std::cout << "Sector Area: " << areaCalculator.calculate_sector_area(M_PI) << std::endl;
    std::cout << "Annulus Area: " << areaCalculator.calculate_annulus_area(2, 3) << std::endl;

    return 0;
}

File: data/derived_cot/rq1_traces/translation/glm5.1/java_to_py/code_output/AreaCalculator.py
import math

class AreaCalculator:

    def __init__(self, radius):
        self.radius = radius

    def calculate_circle_area(self):
        return math.pi * (self.radius ** 2)

    def calculate_sphere_area(self):
        return 4 * math.pi * (self.radius ** 2)

    def calculate_cylinder_area(self, height):
        return 2 * math.pi * self.radius * (self.radius + height)

    def calculate_sector_area(self, angle):
        return (self.radius ** 2) * angle / 2

    def calculate_annulus_area(self, inner_radius, outer_radius):
        return math.pi * (outer_radius ** 2 - inner_radius ** 2)


if __name__ == "__main__":
    area_calculator = AreaCalculator(2)

    print(f"Circle Area: {area_calculator.calculate_circle_area()}")
    print(f"Sphere Area: {area_calculator.calculate_sphere_area()}")
    print(f"Cylinder Area: {area_calculator.calculate_cylinder_area(3)}")
    print(f"Sector Area: {area_calculator.calculate_sector_area(math.pi)}")
    print(f"Annulus Area: {area_calculator.calculate_annulus_area(2, 3)}")